.rubocop_todo.yml

```diff
@@ -0,0 +1,326 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2015-04-27 23:58:52 +0200 using RuboCop version 0.30.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
Not really, you need special config for this project, that's what would go into the original yml file. For example `Style/HashSyntax` set to `EnforcedStyle: hash_rockets`, the auto-gen will not do this for you, it merely disables the cop, which is potentially dangerous. At the moment I have it at the top of the yml file and below is the auto generated stuff.